samples/sid_end_device/sysbuild/mcuboot/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
```

Comment:
the app.overlay and prj.conf file from sdk-mcuboot is needed in sysbuild/mcuboot

Reply:
squashed